src/policy/compiler.rs

```diff
@@ -1396,6 +1395,8 @@ mod tests {
                 vec![],
                 vec![],
                 vec![],
+                vec![],
```

Comment:
. This would bring back to where it was :)

Reply:
Lol, yes.

(FWIW I was not just blindly fixing the test to match the output. I once again had a faulty model of CHECKMULTISIG)